TestFakedWBEMConnection.test_response_delay sometimes fails

The `TestFakedWBEMConnection.test_response_delay()` test asserts that the mocked delay is within certain limits. This fails every now and then.

One example of such a failure is this Travis run: https://travis-ci.org/pywbem/pywbem/jobs/622233290?utm_medium=notification&utm_source=github_status:
```
>           assert delay * .8 < exec_time < delay * 2.0
E           assert 1.006939 < (0.5 * 2.0)
```

--- a/tests/unittest/pywbem_mock/test_wbemconnection_mock.py
+++ b/tests/unittest/pywbem_mock/test_wbemconnection_mock.py
@@ -864,7 +864,8 @@ class TestFakedWBEMConnection(object):
         exec_time = (datetime.now() - start_time).total_seconds()
 
         if delay:
-            assert delay * .8 < exec_time < delay * 2.0
+            assert exec_time > delay * .8
+            assert exec_time < delay * 2.5
         else:
             assert exec_time < 0.1
 

